packages/hint-manifest-icons/src/hint.ts

```diff
@@ -277,13 +347,15 @@ export default class ManifestIconHint implements IHint {
                 if (icons.length > 0) {
 
                     debug(`Validating if manifest-icon file exists`);
-                    const validSizes = await validateIcons(icons, hostnameWithProtocol, resource, getLocation);
+                    const validSizes = await validateIconsSize(icons, hostnameWithProtocol, resource, getLocation);
+                    const iconlocation = getLocation('icons');
 
                     if (validSizes.length > 0) {
-                        const iconlocation = getLocation('icons');
```

Comment:
Seems like this isn't used outside this block, so can probably be kept here rather than moved out.